package.json

```diff
@@ -12,8 +12,29 @@
     "lint": "npm run lint:check",
-    "lint:check": "lerna run lint:check",
-    "lint:fix": "lerna run lint:fix",
+    "lint:check": "npm run lint:check:root && npm run lint:check:core && npm run lint:check:oidc-browser && npm run lint:check:browser && npm run lint:check:node",
+    "lint:fix": "npm run lint:fix:root && npm run lint:fix:core && npm run lint:fix:oidc-browser && npm run lint:check:browser && npm run lint:fix:node",
+    "lint:fix:root": "npm run lint:eslint:root -- --fix && npm run lint:prettier:root -- --write",
+    "lint:check:root": "npm run lint:eslint:root && npm run lint:prettier:root -- --check",
+    "lint:eslint:root": "eslint --config e2e/.eslintrc.js \"e2e\"",
+    "lint:prettier:root": "prettier \"e2e/**/*.{ts,tsx,js,jsx,css}\" \"e2e/**/*.{md,mdx,yml}\" \"./*.{md,mdx,yml}\"",
+    "lint:check:all": "npm run lint:eslint:browser && npm run lint:prettier:browser -- --check",
+    "lint:fix:browser": "npm run lint:eslint:browser -- --fix && npm run lint:prettier:browser -- --write",
+    "lint:check:browser": "npm run lint:eslint:browser && npm run lint:prettier:browser -- --check",
+    "lint:eslint:browser": "eslint --config packages/browser/.eslintrc.js \"packages/browser/src/\"",
+    "lint:prettier:browser": "prettier \"packages/browser/**/*.{ts,tsx,js,jsx,css}\" \"packages/browser/**/*.{md,mdx,yml}\"",
+    "lint:fix:node": "npm run lint:eslint:node -- --fix && npm run lint:prettier:node -- --write",
+    "lint:check:node": "npm run lint:eslint:node && npm run lint:prettier:node -- --check",
+    "lint:eslint:node": "eslint --config packages/node/.eslintrc.js \"packages/node/src/\"",
+    "lint:prettier:node": "prettier \"packages/node/**/*.{ts,tsx,js,jsx,css}\" \"packages/node/**/*.{md,mdx,yml}\"",
+    "lint:fix:core": "npm run lint:eslint:core -- --fix && npm run lint:prettier:core -- --write",
+    "lint:check:core": "npm run lint:eslint:core && npm run lint:prettier:core -- --check",
+    "lint:eslint:core": "eslint --config packages/core/.eslintrc.js \"packages/core/src/\"",
+    "lint:prettier:core": "prettier \"packages/core/**/*.{ts,tsx,js,jsx,css}\" \"packages/core/**/*.{md,mdx,yml}\"",
+    "lint:fix:oidc-browser": "npm run lint:eslint:oidc-browser -- --fix && npm run lint:prettier:oidc-browser -- --write",
+    "lint:check:oidc-browser": "npm run lint:eslint:oidc-browser && npm run lint:prettier:oidc-browser -- --check",
+    "lint:eslint:oidc-browser": "eslint --config packages/oidc-browser/.eslintrc.js \"packages/oidc-browser/src/\"",
+    "lint:prettier:oidc-browser": "prettier \"packages/oidc-browser/**/*.{ts,tsx,js,jsx,css}\" \"packages/oidc-browser/**/*.{md,mdx,yml}\"",
```

Comment:
This is quite a configuration smell, we should probably look at migrating this repository to `nx` and `npm workspaces` instead, which should help here, especially with caching and executing with dependencies.

Reply:
Yup it looks pretty bad, but I'm not sure to what extent this is worse than what we had before: it's the same amount of config, just all in the same place rather than scattered across the monorepo packages.

Reply:
hmm... I feel like we should just be able to run from the root, ignoring lerna & workspaces, and it should just "work"